.github/workflows/release-github.yml

```diff
@@ -5,8 +5,13 @@ on:
     tags:
       - v*
 
+permissions:
+  contents: read
+
 jobs:
   build:
+    permissions:
+      contents: write  # for goreleaser/goreleaser-action to create a GitHub release
```

Comment:
@JeyJeyGao Could you double check this?

Reply:
According to the [document](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents), the content write permission is required for creating a release. Also triggered a release based on the permission setting from my forked repo, and it works fine.